doc/swss-schema.md

```diff
@@ -669,6 +702,21 @@ Status: ready
 
 ## State DB schema
 
+### PORT_TABLE
+Stores information for physical switch ports managed by the switch chip. Ports to the CPU (ie: management port) and logical ports (loopback) are not declared in the PORT_TABLE. See MGMT_PORT.
+
+    ;State for layer 2 ports
+    port_table_key      = PORT_TABLE|ifname    ; ifname must be unique across PORT,INTF,VLAN,LAG TABLES
+    oper_status         = BIT           ; physical status up (1) or down (0) of the link attached to this port
+    state               = BIT           ; netlink ready
+    ifname              = 1*64VCHAR     ; name of the port, must be unique
+
+### MGMT_PORT_TABLE
+    ;State for management port, including at least one key
+    port_table_key      = MGMT_PORT_TABLE|ifname    ; ifname must be unique across PORT,INTF,VLAN,LAG TABLES
+    oper_status         = BIT           ; physical status up (1) or down (0) of the link attached to this port
+    ifname              = 1*64VCHAR     ; name of the port, must be unique, eg. eth0
+
```

Comment:
do we need to have LOOPBACK_PORT_TABLE?

Reply:
loopback is not a port. We may consider it later.